src/ui/public/field_editor/field_editor.js

```diff
@@ -129,6 +154,14 @@ uiModules
         else return fieldFormats.getDefaultType(self.field.type);
       }
 
+      function getScriptingLangs() {
+        return $http.get(chrome.addBasePath('/api/kibana/scripts/languages'))
+        .then((res) => res.data)
+        .catch(() => {
+          return $q.reject(new Error('Error getting available scripting languages from Elasticsearch'));
```

Comment:
Actually, those errors seem to be tailored to handling responses from the Elasticsearch proxy in Kibana. Using it with the response from this Kibana API gives a nasty "unknown" or "undefined" error, so I'm going to stick with with this custom message that's more user friendly. However I did update it to use the notifier that's already configured for this directive instead of simply rejecting with a new error.